src/Rel8/Category/Subcolumns.hs

```diff
@@ -0,0 +1,67 @@
+{-# language DataKinds #-}
+{-# language ExistentialQuantification #-}
+{-# language MultiParamTypeClasses #-}
+{-# language StandaloneKindSignatures #-}
+{-# language TypeFamilies #-}
+
+module Rel8.Category.Subcolumns
+  ( Subcolumns( Subcolumns )
+  )
+where
+
+-- base
+import Control.Category ( Category, (.), id )
+import Data.Kind ( Type )
+import Prelude hiding ( (.), fst, id, snd )
+
+-- categories
```

Comment:
Maybe I'm blind, but I can't see a `categories` dependency in `build-depends`. I also note that this module isn't in `other-modules` or `exposed-modules`, so maybe this is an experiment you didn't mean to commit?

Reply:
I was pretty sure I'd removed this file from this branch

Reply:
But yeah, you're right. Removed it now!